### DISCUSSION TITLE
Ubuntu upgrade included GO upgrade ;-)

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1183] [General]
I have just upgraded Ubuntu from 20.04 LTS to 22.04 LTS. I did this from a shell (terminal) using the command sudo do-release-upgrade. When testing after upgrade I found that all MIDI settings were lost. To my surprise, GO had also been upgraded to 3.6.2 without my noticing it.
After mapping MIDI events to manuals, pedals and controls on my FCB1010 it all worked as before :-)
Alas, a small word of warning fot those who have not planned the GO upgrade to happen when upgrading Ubuntu.

//Jens

**Comment by jason-barba** [comment]
Funny though. On my sandbaox I got GO version 3.6.7-1 (upgraded with the GUI tool). On the live system, upgraded using CLI command I got GO version 3.6.2.
Any comments on that?

**Comment by oleg68** [comment]
When you upgrade your OS the all packages available in the standard repositories are upgraded as well by default. You can add some packages to the blacklist for preventing their automatic upgrade.

Please ask the GrandOrgue packgage mainantainer of your distro for details about the certain version: https://packages.ubuntu.com/jammy/grandorgue
